.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,15 +9,15 @@ Make sure to point your PR to the relevant develop branches, e.g.
 How to contribute: https://contribute.getkirby.com
 -->
 
-### What has changed?
+### This PR changes…
 
 
 
-### Why was it changed?
+### Reason
 
 
 
-### What should reviewers know?
+### For reviewers
```

Comment:
What were your thoughts behind making these shorter? TBH I feel they are harder to understand now.

Reply:
I found it hard to read that first section if all headlines are so similar (What, Why, What...). IMO we should make them easy to parse, even at a glance.

Reply:
```suggestion
### Additional context
```